musictheorpy/notes.py

```diff
@@ -63,6 +63,21 @@ def descend_interval(self, qualified_interval_name):
             raise NoteNameError("Descending a %s from %s results in an invalid note." % (qualified_interval_name,
                                                                                          self._qualified_name)) from None
 
+    def find_interval_from_root(self, top_note):
+        """
+        :param top_note: a string that is verified as a valid Note and passed to the
+        interval_builder.find_interval_from_root method to find its relation to root note
+        :return <str> interval: the interval at which the top note is in relation from the root
+        """
+        root_note = self.qualified_name
```

Comment:
This assignment is not necessary. Please access self.qualified_name where you need it to remove redundancy.